web/beacon-app/src/components/ui/SetupTenant/CustomizeTenant.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import { Meta, Story } from '@storybook/react'
```

Comment:
I think the setupTenant components should be moved to `@features/onboarding/components`